news/2020-31-12-mappers-report-fall.md

```diff
@@ -0,0 +1,265 @@
+
+The next is a pair of recent videos from long time mapper and Beatmap Nominator [Sonnyc](https://osu.ppy.sh/users/11771). The first, [Making Symmetric Sliders](https://www.youtube.com/watch?v=nYkMFvq4ZLU) serves to quickly show you how to make symmetrical sliders quickly. Even if you don't map symmetrically, knowing how to make balanced sliders to manipulate later can be hugely helpful. The second, [Increasing Mapping Speed](https://www.youtube.com/watch?v=sMKJ9H-qGg0) goes over an experience nearly every mapper has had - mapping slowly at first, and then later how to improve on putting your ideas out there quickly. Other videos on Sonnyc's channel can be super helpful guides to mappers new and old alike, so be sure to check them out!
+
+Moving away from motivation, time taken, and visuals, we have a [guide all about hitsounding](https://hackmd.io/@Net0/HitsoundsGuide) by [Net0](https://osu.ppy.sh/users/5099768)! This guide about hitsounds is incredibly in-depth, going all the way from how to name your files to theory, choosing samples, using the editor, and how hitsounding relates back to actual instruments in the songs we love to map and play. If you've ever found hitsounding beatmaps to be intimidating, this guide will definitely help to clear up a lot of questions about it you may have.
+
+Finally, on a bit of a different flavor we have a guide not related to mapping - but modding! The [Modding Mentorship Guide](https://sites.google.com/view/modding-guide/home) written by long time mapper and nominator [Cheri](https://osu.ppy.sh/users/5226970) seeks to help give guidance on how to mod and what to look out for from the ground up. This guide covers everything from ranking criteria, to rhythm, to spacing, to low difficulties, and spread. It covers quite a lot! In-depth modding guides of this kind are a bit rarer and left to people often learning on their own, so having a fully fleshed out guide like this will definitely be really helpful to aspiring and semi-experienced modders alike.
+
+<a id="conclusion"></a>
+
+---
+
+And that concludes the last issue of the Mappers' Report in the year 2020! Quite a lot to read again, but we are happy should you have survived until here.
+
+To the end of the year, the lead of the Mappers' Report [Feerum](https://osu.ppy.sh/users/4815717), would like to drop a few words: 
+
+The Mappers' Report started as a project together with the Beatmap Nominators out of a whim after a NAT Meeting. First we NAT just wanted to use this as a plattform to reach more participants for the meetings and inform these who couldn't participate about what exactly was discussed. This worked by the way! 
+But over the time i have gotten more ideas for the whole project and it became huge! I added more cathegories because i wanted to bring as much informations about the, in my eyes, most important aspect in osu! to as many people as possible. 
```

Comment:
capitalise your i's feerum

"over ~~the~~ time"

categories

"as much information about the most important aspects of osu! in my eyes"